### ISSUE TITLE
Unleash a new dashboard UI in 2.7

### ISSUE THREAD

**Issue by g1eny0ung** [chaos-mesh/chaos-mesh#4098]
After my experimental changes, our UI interface needs to eliminate some useless things, like predefined experiments on the /dashboard page. It rarely has usage in the past. Or like global searching, It is also rarely used.

I prepare to cut these functions and make the dashboard to be more clear to focus on core features. And I wanted to update the previous design in the interface, which I should have done a long time ago, but I procrastinated. 😩 I may update some components' styles in the previous design due to our features are also evolving, so just take it as a reference: https://www.figma.com/file/2J6PVAaitQPQFHBtF5LbII/UI-Interface?type=design&node-id=0%3A1&mode=design&t=jkMWgPPPxk3L105T-1

Below are some of my works in the last week (for a preview):

<img width="1410" alt="image" src="https://github.com/chaos-mesh/chaos-mesh/assets/15034155/49764021-8b20-419c-ba63-43511ae9744e">

<img width="1415" alt="image" src="https://github.com/chaos-mesh/chaos-mesh/assets/15034155/4f18c945-104e-4854-a990-c40aba6f291c">

**Comment by victoriacheng15** [comment]
@g1eny0ung, checking in on this. The whiteboard UI was integrated in PR #4698 and released in PR #4699. This issue could be closed. 😄

**Comment by g1eny0ung** [comment]
@victoriacheng15 Yes, those two merged PRs are indeed part of the progress on this PR, but they don't cover everything. I had originally planned to refactor the entire UI, but I’ve been slacking off... Thanks for the reminder. Now I’m planning to use AI to help me get the job done. 🦾

**Comment by victoriacheng15** [comment]
> [@victoriacheng15](https://github.com/victoriacheng15) Yes, those two merged PRs are indeed part of the progress on this PR, but they don't cover everything. I had originally planned to refactor the entire UI, but I’ve been slacking off... Thanks for the reminder. Now I’m planning to use AI to help me get the job done. 🦾

Gotcha! Thank you for the updated info! 😃
